src/components/collections/ProgramPackageCollection.tsx

```diff
@@ -215,6 +216,7 @@ const composeCollectionData = (data: hasura.GET_PROGRAM_PACKAGE_COLLECTION): Pro
       listPrice: ppp.list_price,
       salePrice: ppp.sale_price,
       soldAt: ppp.sold_at && new Date(ppp.sold_at),
+      publishedAt: ppp.published_at,
```

Comment:
FIX: should transform to date if not null, because the type you definition is date or null
like above soldAt